src/javascript/app/ng-wig/ng-wig.js

```diff
@@ -3,5 +3,5 @@
  */
 angular.module('ngWig', ['ngwig-app-templates']);
 angular.ngWig = {
-  version: '3.0.1'
+  version: '3.0.4'
```

Comment:
it's better to update version together with a build, so could you please remove it from this PR?